From 31e37eda2b5724edfa4e26f34dc91701be0bb94c Mon Sep 17 00:00:00 2001
From: Yann Gautier <yann.gautier@foss.st.com>
Date: Tue, 24 Oct 2023 16:36:58 +0200
Subject: [PATCH 406/841] refactor(st-clock): remove DDR clock init

It is now done in stm32mp2_ddr_helpers.c file.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 8c4d3de65178fdc994cc939afc26acfd40c306ff refactor(st-clock): remove DDR clock init]
Change-Id: Icca767bdb8901c3c36eaad5d5f0b92d97d712275
Signed-off-by: Yann Gautier <yann.gautier@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/335915
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
---
 drivers/st/clk/stm32mp2_clk.c | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/drivers/st/clk/stm32mp2_clk.c b/drivers/st/clk/stm32mp2_clk.c
index 0d130ae39..4df5051de 100644
--- a/drivers/st/clk/stm32mp2_clk.c
+++ b/drivers/st/clk/stm32mp2_clk.c
@@ -2287,12 +2287,6 @@ static int stm32_clk_parse_fdt(struct stm32_clk_platdata *pdata)
 }
 #endif /* IMAGE_BL2 */
 
-static void stmp32mp2_clk_init_ddr_sub_system(uintptr_t base)
-{
-	mmio_write_32(base + RCC_DDRCPCFGR,
-		      RCC_DDRCPCFGR_DDRCPEN | RCC_DDRCPCFGR_DDRCPLPEN);
-}
-
 static struct stm32_osci_dt_cfg mp25_osci[NB_OSCILLATOR];
 
 static struct stm32_pll_dt_cfg mp25_pll[_PLL_NB];
@@ -2355,8 +2349,6 @@ int stm32mp2_clk_init(void)
 		return ret;
 	}
 
-	stmp32mp2_clk_init_ddr_sub_system(base);
-
 #ifdef IMAGE_BL2
 	ret = stm32mp2_init_clock_tree();
 	if (ret != 0) {
-- 
2.47.2

